epel8/openscap-report.rhel8.spec

```diff
@@ -0,0 +1,64 @@
+%global pymodule_name openscap_report
+
+Name:           openscap-report
+Version:        0.0.0
+Release:        0%{?dist}
+Summary:        A tool for generating human-readable reports from (SCAP) XCCDF and ARF results
+
+# The entire source code is LGPL-2.1+ and GPL-2.0+ and MIT except schemas/ and assets/, which are Public Domain
+License:        LGPLv2+ and GPLv2+ and MIT and Public Domain
+URL:            https://github.com/OpenSCAP/%{name}
+Source0:        https://github.com/OpenSCAP/%{name}/releases/download/v%{version}/%{name}-%{version}.tar.gz
+
+BuildArch:      noarch
+%if 0%{?rhel} == 8
```

Comment:
If this spec file is only for RHEL 8 this test is useless. If the spec file is more generic (RHEL 9 as well, for example) it should be renamed. I'd strongly recommend somehow integrate it into the main spec file, if possible.

Reply:
Yes, definitely you are right. That test is there because of the Packit. Somehow Packit builds a package with an RHEL8 spec file for all Fedora versions which causes false positive results of tests. Maybe it is Packit's bug. I will try to investigate where is a problem. Maybe the RHEL8 rpmbuid test is not needed in upstream.  

I don't know if the integration of this spec file to the main spec file is a good idea because RHEL8 supports macros from [201x-era Python guidelines](https://docs.fedoraproject.org/en-US/packaging-guidelines/Python_201x/). So there will be more ugly RHEL8 tests.